### DISCUSSION TITLE
Where is `reactive-rpc`?

### DISCUSSION THREAD

**Discussion by gabewillen** [streamich/json-joy#678] [Q&A] [answered]
Why was Reactive-RPC removed from json-joy?

**Comment by streamich** [comment, marked as answer]
The `reactive-rpc` sub-library is now its own standalone library at [`jsonjoy-com/reactive-rpc`](https://github.com/jsonjoy-com/reactive-rpc). It was not "removed" per se, but it grew in size and is a distinctive networking piece, which only made sense to promote to its own package.

```
npm install @jsonjoy.com/reactive-rpc
```

**Reply by gabewillen** [reply]
Awesome! Ya I saw it on npm but the GitHub link is broken so I was wondering if you were taking it closed source. I also searched GitHub for it and went a few pages in with no luck. It definitely makes sense to pull it out of json-joy. Thanks for all of your open source contributions!

Cheers
